src/Helper.js

```diff
@@ -0,0 +1,10 @@
+export function getRandomUniqueNumbersArray(max) {
+    const randomNumArr = [];
+    while (randomNumArr.length < max) {
+        const random = Math.floor(Math.random() * 5);
```

Comment:
```suggestion
        const random = Math.floor(Math.random() * max);
```
Здається тут треба max